homeworks/ekaterina.sharova_rogerosha/PokemonProject/index.html

```diff
@@ -5,27 +5,49 @@
         <meta name="viewport"
             content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
         <meta http-equiv="X-UA-Compatible" content="ie=edge">
-        <title>PokemonProject</title>
+        <title>PokemonProject: About</title>
         <link
             href="./style.css"
             rel="stylesheet"
             type="text/css">
     </head>
     <body>
-        <h1 class="headline">About me</h1>
-        <p>
-            Hi! My name is Katya and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like
-            HTML, CSS, JavaScript, and Git version control system.
-        </p>
-        <p>
-            This page was developed during the course <a href="https://www.mastersacademy.education/frontend-for-beginners-it" class="link"> 'Frontend for beginners' from Masters Academy in 2023</a>.
-        </p>
-        <p>
-            This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my
-            own small project for the portfolio.
-        </p>
-        <p>
-            You can contact me via telegram @rogerosha and/or check out my GitHub.
-        </p>
+        <header class="upper-header">
+            <div class="ma-logo">
+                <img class="MAlogo" src="MAlogo.png" alt="Masters Academy">
```

Comment:
Hey, @rogerosha, better to export the logo as one piece than break it in two. And always export svg format if it is a vector on the design
Also, the px size should be an integer
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/ad52e8a9-4f31-4da2-a253-e6beda624ca3)

Reply:
Hello, Vlad, thank you for the advice! It's a really good decision for export and import